util/topgen/templates/plic_all_irqs_test.c.tpl

```diff
@@ -5,6 +5,8 @@
 ${gencmd}
 <%
 irq_peripheral_names = sorted({p.name for p in helper.irq_peripherals})
+has_csrng = "csrng" in irq_peripheral_names
```

Comment:
Nit: It might be shorter to just use `csrng_index` (from the next line) and change the tests below to something more like `% if csrng_index is not None:`

Reply:
On the other hand I find `if has_csrng` is preferable to `if csrng_index is not None`. I'll pass on this one.